CMakeLists.txt

```diff
@@ -1,10 +1,15 @@
 # CMakeLists.txt has to be located in the project folder and cmake has to be
 # executed from 'project/build' with 'cmake ../'.
-cmake_minimum_required(VERSION 2.6)
+cmake_minimum_required(VERSION 3.1)
```

Comment:
This should be at least 3.5 to avoid deprecation warnings.

Reply:
Hmm. Right, we don't really need to target anything older than Ubuntu 16.04(which is the first that comes with cmake 3.5). Thanks, will further change that.